stable/jenkins/README.md

```diff
@@ -482,6 +483,40 @@ master:
 Docs taken from https://github.com/jenkinsci/docker/blob/master/Dockerfile:
 _Jenkins is run with user `jenkins`, uid = 1000. If you bind mount a volume from the host or a data container,ensure you use the same uid_
 
+## Adding custom pod templates
+It is possible to add custom pod templates for the default configured kubernetes cloud.
+Add a key under `agent.podTemplates` for each pod template. Each key (prior to | character) is just a label, and can be any value.
+Keys are only used to give the pod template a meaningful name.  The only restriction is they may only contain RFC 1123 \ DNS label
+characters: lowercase letters, numbers, and hyphens. Each pod template can contain multiple containers.
+There's no need to add the *jnlp* container since the kubernetes plugin will automatically inject it into the pod.
+For this pod templates configuration to be loaded the following values must be set:
+```
+master.JCasC.enabled: true
+master.JCasC.defaultConfig: true
+master.JCasC.configScripts: at least one key (cannot be empty {})
+```
+The example below creates a python pod template in the kubernetes cloud.
+```
+agent:
+  podTemplates:
+    python: |
```

Comment:
FYI. I noticed the [JenkinsX values file](https://github.com/jenkins-x/jenkins-x-platform/blob/731a37428e057501e294d2737ebff785052e1fcf/jenkins-x-platform/values.yaml#L1057) doesn't use multi-line strings. As it turns out, having this be a multi-line string (and not calling `toYaml` in the template) actually provides additional flexibility and allows for passing in a block of templating.

```
agent:
  podTemplates:
    agents: |-
      {{- /* save chart agent as `set` modifies the input dictionary */}}
      {{- $rootAgent := .Values.agent }}

      {{- /* each additional agent inherits from $rootAgent */}}
      {{- range .Values.additionalAgents }}
      {{- include "jenkins.casc.podTemplate" (set $ "Values" (set $.Values "agent" (merge . $rootAgent))) }}
      {{- end }}

      {{- /* restore chart agent */}}
      {{- $_ := set .Values "agent" $rootAgent }}
```

Reply:
I'm glad you pointed me this out and resulted to be a good thing. I just followed examples from other parts of this chart.
So, does it looks already ok to test to you? 
It's my first contribution here, how long does this process tend to take?